bucket/rakudo-star.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://rakudo.org/",
+    "version": "2020.01-01",
+    "license": "Artistic-2.0-Perl",
+    "description": "The most mature, production-ready implementation of the Raku language",
+    "url": "https://rakudo.org/dl/star/rakudo-star-2020.01-01-win-x86_64-(JIT).msi",
+    "hash": "A4A728FFF7C32D98790E4E12F145BFAB89CC4DAB56E268B2CED3C80F08BD4A30",
+    "extract_dir": "rakudo",
+        "bin": [
+        "bin\\perl6.exe",
+        "bin\\raku.exe",
+        "bin\\rakudo.exe"
+    ],
+    "checkver": {
+        "url": "https://rakudo.org/downloads/star",
+        "regex": "rakudo-star-([\\d]{4}.[\\d]{2}-[\\d]{2})-win-x86_64-\\(JIT\\).msi"
```

Comment:
```suggestion
        "regex": "rakudo-star-([\\w.-]+)-win-x86_64-\\(JIT\\)\\.msi"
```

Reply:
would the '-' in '-]+' not result in '2020.01-' instead of '2020.01-01' (has no "one or more - at the end of the match")?

Reply:
No. Why it should?

`[]` is character class. It is just to clarify what possible characters could appear in sequence. current sequence could contain numbers, dots and dash.

Just visualize it in regex101.